src/celeritas/io/ImportParameters.hh

```diff
@@ -17,6 +17,24 @@
 
 namespace celeritas
 {
+//---------------------------------------------------------------------------//
+/*!
+ * User-imposed cutoffs and restrictions.
+ *
+ * Used as tracking cuts, see \c G4UserLimits , used by \c G4UserSpecialCuts.
+ * As "production" cuts, only for electron/positron/proton/gamma, see \c
+ * G4ProductionCuts .
+ */
+struct ImportCutoff
```

Comment:
I'm pondering about this name. This is another set of cuts beyond the `ImportProductionCuts` which originally are loaded into our `CutoffParams`---which bears a strikingly similar name to this one. So maybe we need that to be `ProductionCutParams` instead?

Reply:
I guess these are tracking cuts; I think we should combine them with production cuts in the codebase. I've been trying to untangle the particle production cuts, regions, "cut couple", etc. I forgot that I didn't actually implement the reading for these cuts from G4UserLimits .... I'll just remove them for this PR and we can add them to an "extended cut" set later. We should also determine how to make the `apply_cuts` and `lowest_electron_energy` EM parameters consistent with these options.

Reply:
Yep, you reached another question I had while going over it: that this `ImportCutoff` struct is not being used yet. Okay, that sounds good to me. And later on, I think it makes sense to keep a `CutoffParams` that encompasses both production cuts and extended cuts by adding both to the `CutoffParamsData`.